hedera-node/hedera-app/src/main/java/com/hedera/node/app/Hedera.java

```diff
@@ -633,6 +633,7 @@ private void migrateSchemas(
         }
         blockStreamService.resetMigratedLastBlockHash();
         startupNetworks = startupNetworksFactory.apply(selfNodeId.id(), configProvider, tssBaseService);
+        PLATFORM_STATE_SERVICE.setAppVersionFn(() -> version);
```

Comment:
This isn't necessary since the version can be [directly computed](https://github.com/hashgraph/hedera-services/blob/16551-startup-assets/hedera-node/hedera-app/src/main/java/com/hedera/node/app/version/ServicesSoftwareVersion.java#L81) from the `Configuration` on the `MigrationContext`,
```
ServicesSoftwareVersion.from(ctx.configuration())
```